.travis.yml

```diff
@@ -26,9 +26,6 @@ install:
 script:
   - npm run-script coverage
 
-after_script:
-  - cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js
```

Comment:
Not hat was probably just a mistake on my part. I changed the write permissions to admin permissions.

<img width="783" alt="screen shot 2017-02-04 at 4 24 05 pm" src="https://cloud.githubusercontent.com/assets/542335/22621644/653d477e-eaf6-11e6-937a-fc8cdc8d526c.png">